src/ui/CategoryFacet/CategoryFacetValues/CategoryFacetValueRenderer.ts

```diff
@@ -0,0 +1,32 @@
+import { $$, Dom } from '../../../utils/Dom';
+import { CategoryFacet } from '../CategoryFacet';
+import { CategoryFacetValue, ValueRenderer } from './CategoryFacetValue';
+
+export class CategoryFacetValueRenderer implements ValueRenderer {
+  private dom: Dom;
+
+  constructor(private facetValue: CategoryFacetValue, private facet: CategoryFacet) { }
+
+  public render() {
+    this.dom = $$('li', {
```

Comment:
Should we add certain accessibility attributes to each li? If so, this could be included here on in a separate PR focused on accessibility.

Reply:
Yes it's basic displaying for now and I will add all the necessary style + accessibility in another PR